daffodil-japi/src/test/java/org/apache/daffodil/example/TestJavaAPI.java

```diff
@@ -931,6 +931,15 @@ public void testJavaAPI20() throws IOException, ClassNotFoundException {
         assertEquals("42", unparseBos.toString());
     }
 
+    @Test
+    public void testJavaAPINullXmlOutputStyle() {
+        ByteArrayOutputStream xmlBos = new ByteArrayOutputStream();
+        try {
+            XMLTextInfosetOutputter outputter = new XMLTextInfosetOutputter(xmlBos, true, null);
+        } catch (Exception e) {
+		  assertTrue(e.getMessage().contains("Null is not a valid value for parameter xmlOutputStyle"));
+        }
+    }
```

Comment:
Daffodil contributors are expected to maintain the Daffodil codebase's standard Scala formatting (2 spaces per indentation, etc.).  Your new test (thank you for the test!) is formatted with 4 spaces per indentation and also contains 2 tabs on line 940.  I suggest you configure your IDE with standard Scala formatting settings (including no tabs in indentation) and then use your IDE to indent your new test.  DEVELOP.md has a section on code style and an IDEA setup link.  We don't have sbt set up to run a formatter over the Daffodil codebase, so we have to rely on PR review to keep the codebase's formatting consistent.

Also, consider testing only that the message contains the word "xmlOutputStyle".  Otherwise, the test will be fragile and break as soon as someone changes that exception's message even slightly (rearrange the order of words, lowercase Null, etc.).

Reply:
I saw this wasn't fixed yet, so added a comment on the new line of code where this problem still exists.
So resolving this here.